### ISSUE TITLE
Add Name of the Prüfidentifikator Dropdown list

### ISSUE THREAD

**Issue by hf-krechan** [Hochfrequenz/ahb-tabellen#160]
and make it also searchable.

The user would like to see also the name of the Prüfidentifikator, e.g. `55044 - Anmeldung MSB` instead of only `55044`.

The user would also like to search for the name of the Prüfidentifikator.
The search should also have the forward typing kind of search.
If the user starts typing `Anmeldung` than all prüfis which starts or contain this word should be visible.

Maybe we need than to discuss a new architecture setup to achieve this goal.

**Comment by hf-krechan** [comment]
The information of the name is already in the json files in the `meta` section.

**Comment by hf-kklein** [comment]
https://github.com/Hochfrequenz/machine-readable_anwendungshandbuecher/blob/20000e7c8725c2f320f505fca1841e81b3685543/FV2504/UTILMD/flatahb/55044.json#L1084

**Comment by hf-kklein** [comment]
this is definitly a high prio issue for users

**Comment by OLILHR** [comment]
might require something like a mapping prior to accessing files so the server is able find pruefIDs by their name (currently only provided by the metadata inside the AHB files)

**Comment by hf-kklein** [comment]
discussed with @hf-krechan : we postponed this, because it's foreseeable, that once you start with anything else but the prüfi, users will request stuff like: 
- but i want to search for the name
- but i want to search for the role
- but i want to search ´<anything else from inside the ahb>`

and that's where you file-base approach isn't the best choice anymore. so we'll first try to load the machine-readable AHBs (be it the jsons or the XMLs) into a db, and then go on from there. but it's out of scope for the first prod release.

**Comment by hf-kklein** [comment]
we did it!

![Image](https://github.com/user-attachments/assets/8cd81285-d23d-4c2a-bc42-b57433f68e0b)

@hf-krechan ich denke, das kann zu?

**Comment by hf-krechan** [comment]
yes kann es. Danke deiner guten Idee mit der sqlite Datenbank :)
